pkg/apis/authentication/v1alpha1/types_viewerkubeconfigrequest.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
One small nit, I missed.
```suggestion
Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Reply:
That nit is very expensive 💸